pyproject.toml

```diff
@@ -18,6 +18,8 @@ pydantic = "^1.10.18"
 poetry-core = "^1.9.1"
 pyOpenSSL = "^24.2.1"
 jinja2 = "^3.1.4"
+pysyncobj = "^0.3.13"
```

Comment:
We should most probably bump in the snap as well, [though changes appear minor](https://github.com/bakwc/PySyncObj/releases/tag/v0.3.13).